src/Unleash/Strategies/Constraints/SemverConstraintOperator.cs

```diff
@@ -0,0 +1,72 @@
+﻿using NuGet.Versioning;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Unleash.Internal;
+using Unleash.Logging;
+
+namespace Unleash.Strategies.Constraints
+{
+    public class SemverConstraintOperator : IConstraintOperator
+    {
+        private static readonly ILog Logger = LogProvider.GetLogger(typeof(SemverConstraintOperator));
+
+        public bool Evaluate(Constraint constraint, UnleashContext context)
+        {
+            var contextValue = context.GetByName(constraint.ContextName);
+            SemanticVersion contextSemver;
+            if (!SemanticVersion.TryParse(contextValue, out contextSemver))
+            {
+                Logger.Info("Couldn't parse version {0} from context");
+                return false;
+            }
+
+            if (constraint.Values != null && constraint.Values.Any())
```

Comment:
After checking the spec and checking with @chriswk it's decided that i should switch to only .Value here

Reply:
:+1: